HelpMyStreetFE/HelpMyStreetFE/Views/Community/FavershamAndSittingBourne.cshtml

```diff
@@ -5,7 +5,7 @@
 
 @{
     ViewData["Title"] = Model.CommunityName;
-    ViewData["HMSLogoViewModel"] = new HMSLogoViewModel() { LinkEntireLogo = false, PrimaryText = "Faversham And Sitting Bourne", PrimaryMobileText = "Faversham And Sitting Bourne", ShowSecondaryText = true };
+    ViewData["HMSLogoViewModel"] = new HMSLogoViewModel() { LinkEntireLogo = false, PrimaryText = "Faversham And Sittingbourne", PrimaryMobileText = "Faversham And Sittingbourne", ShowSecondaryText = true };
```

Comment:
Please can you do a find and replace on "Sitting Bourne"?  There are a few other instances on this page.